src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -546,56 +546,86 @@ ucs_status_t uct_tcp_sockcm_ep_send(uct_tcp_sockcm_ep_t *cep)
     return uct_tcp_sockcm_ep_progress_send(cep);
 }
 
+static ucs_status_t
+uct_tcp_sockcm_ep_get_remote_device_addr(const uct_tcp_sockcm_ep_t *cep,
+                                         struct sockaddr_storage *saddr,
+                                         socklen_t *saddr_len_p,
+                                         uct_tcp_device_addr_t **
+                                                 remote_dev_addr_p,
+                                         size_t *remote_dev_addr_len_p)
```

Comment:
maybe caller can pass (large onstack) buffer and max size; this function will fill it and return ssize_t

Reply:
done